pom.xml

```diff
@@ -12,7 +12,21 @@
 
 	<properties>
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+		<!-- tomcat version - version 7 recommended, supports java 1.6+ -->
+		<!-- tomcat jars need to be in the repo though -->
 		<tomcat.version>6.0.26</tomcat.version>
+		<!-- 
+		<tomcat.version>7.0.67</tomcat.version>		
+		 -->
+		 <!-- tomcat version -->
+		<openmrs.version>2.0.0-SNAPSHOT</openmrs.version>
+		<!-- use -D to specify the os_arch to select mysqld -->
+		<!-- 
+		<os.name>Win</os.name>
+		<os.arch>x86</os.arch>
+		 -->
+		<!-- os_arch -->
+		<javaVersion>1.8</javaVersion>
 	</properties>
```

Comment:
Right now we don't use any new APIs in  java 8, so it fine to leave the target and source versions of the compiler at 1.6, i think the goal is to make it work for platform 2.0 and java 8